Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1755,6 +1756,7 @@ def return_error(message, error='', outputs=None):
         :return: Error entry object
         :rtype: ``dict``
     """
+    message = "{}\n\nTrace:{}".format(message, traceback.format_exc())
```

Comment:
`format_exc()` already returns some thing like `Traceback (most recent call last):` no?
if so, lets remove the string `Trace:` from here